src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -161,7 +161,7 @@ export const generateNavItems = (): NavItem[] => {
     {
       id: 'settings',
       testID: 'nav-item-settings',
-      icon: IconFont.WrenchNav,
+      icon: IconFont.CogOutline_New,
```

Comment:
Same question here - looks like it's not just `CogOutline` that is getting replaced with `CogOutline_New`.